### DISCUSSION TITLE
1.1版本的doris 使用java udf功能创建function提示unknown function type

### DISCUSSION THREAD

**Discussion by zhoulong0517** [apache/doris#11829] [RFC]
doris1.1版本官网说，可以使用java udf
编写 UDF 函数
本小节主要介绍如何开发一个 Java UDF。在 samples/doris-demo/java-udf-demo/ 下提供了示例，可供参考，查看点击[这里](https://github.com/apache/incubator-doris/tree/master/samples/doris-demo/java-udf-demo)

示例：

CREATE FUNCTION java_udf_add_one(int) RETURNS int PROPERTIES (
    "file"="file:///path/to/java-udf-demo-jar-with-dependencies.jar",
    "symbol"="org.apache.doris.udf.AddOne",
    "type"="JAVA_UDF"
);
用1.1版本的doris创建这个fuction（使用官网例子创建），提示一下错误信息：
mysql> CREATE FUNCTION java_udf_add_one(int) RETURNS int PROPERTIES (
    ->     "file"="file:///path/to/java-udf-demo-jar-with-dependencies.jar",
    ->     "symbol"="org.apache.doris.udf.AddOne",
    ->     "type"="JAVA_UDF"
    -> );
ERROR 1105 (HY000): errCode = 2, detailMessage = unknown function type

**Comment by dataalive** [comment]
it will support on 1.2. we will submit a pr to fix this document typo

**Reply by zhoulong0517** [reply]
I really appreciate your help,thanks so much.

**Reply by zhoulong0517** [reply]
> it will support on 1.2. we will submit a pr to fix this document typo

Hi，
thanks for your help, if the issue fixed on 1.2, cause the 1.2 has been published

**Comment by weiatengland** [comment]
I'm still getting this "unknown function type" error, has this been fixed?

**Reply by zy-kkk** [reply]
Java UDF is supported by version 1.2, please upgrade to use, if you still have problems in 1.2, please feedback
